src/app.js

```diff
@@ -147,11 +145,19 @@ async function connectEngine(appName) {
         } else {
           console.error(error);
         }
+        return error;
       },
     }],
   });
   const qix = await session.open();
-  const app = await qix.openDoc(appName);
+  let app;
+  try {
+    app = qix.openDoc(appId);
+  } catch (err) {
+    if (err.code === schemaEnigma.enums.LocalizedErrorCode.LOCERR_APP_ALREADY_OPEN) {
+      app = qix.getActiveDoc();
+    }
```

Comment:
Why not check the process.env.NODE_ENV to decide whether to use openDoc or getActiveDoc? Works either way ofc :)

Reply:
yes - locally (docker-compose) the app can't do `getActiveDoc` since there is no session app involved like in the core-scaling backend